Sources/Display.php

```diff
@@ -864,7 +864,7 @@ function Display()
 	$ascending = empty($options['view_newest_first']);
 
 	// Check if we can use the seek method to speed things up
-	if (isset($_SESSION['page_topic']) && $_SESSION['page_topic'] == $topic)
+	if (isset($_SESSION['page_topic']) && $_SESSION['page_topic'] == $topic && $_SESSION['page_ascending'] == $ascending)
```

Comment:
> I'm not 100% sure if a isset check should be placed too,
> normaly the var should be setted in the same moment when $_SESSION['page_topic'] is setted.
> So when $_SESSION['page_topic'] is not setted the $_SESSION['page_ascending'] var should be also not setted and
> vice versa.

It would be best to include `isset($_SESSION['page_ascending'])`, just as a precaution. Without it, bored troublemakers could flood a forum's error log by manipulating their browser's stored session variables.

Reply:
When you like I can do it,
But your mention way doesn't work. 
The session is saved on server side, 
When your are evil you switch your session id but then you got complete different package of seib vars.

Reply:
🤦‍♂️ I'm not sure what I was thinking there. Of course that's correct. Never mind what I said before. 🙂